hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -329,6 +329,14 @@ private FlinkOptions() {
       .withDescription("Whether to skip clustering instants to avoid reading base files of clustering operations for streaming read "
           + "to improve read performance.");
 
+  // this option is experimental
+  public static final ConfigOption<Boolean> READ_STREAMING_SKIP_INSERT_OVERWRITE = ConfigOptions
+      .key("read.streaming.skip_insertoverwrite")
+      .booleanType()
+      .defaultValue(false)
```

Comment:
Should the default value be true?

Reply:
This is different from clustering and compaction. Skipping clustering and compaction will not cause inconsistency in the upstream and downstream data in the streaming reading scenario. By default, the upstream and downstream data should be consistent.  Developers must make their own decisions knowing that this will lead to inconsistency in the upstream and downstream data， and then enable this feature as needed.